### PR TITLE
added exercise

### PR DESCRIPTION
**Pull request by ChristopherBittner**

### CODE REVIEW

File: 000/0001/tst.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                correct = True
+                break
+        if correct:
+            dividers.append(i)
+
+    return dividers
+
+ret = find_natural_dividers(test_input)
+print("Function is working {} - {}.".format("correct" if ret == test_output else "incorrect", ret))
+print("Function usage for 1000 is {}".format(find_natural_dividers(1000)))
```
Comment (EwaBittner) [inline 000/0001/tst.py]: Why don't you leave the last line empty?

File: 000/0001/tst.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    for i in range(1, val):
+        correct = False
+        for mult in MULTIPLIERS:
+            if i % mult == 0:
+                correct = True
+                break
+        if correct:
+            dividers.append(i)
+
+    return dividers
```
Comment (EwaBittner) [inline 000/0001/tst.py:22]: Where is the line showing the result?

**Review by EwaBittner — COMMENTED** [review]
Quite hard for me to see what is actually happening here. I think running the code in Python would make it easier to see.

File: 000/0001/tst.py (hunk 2 above)
Comment (ChristopherBittner) [reply 000/0001/tst.py:22]: Used directly in prints.

File: 000/0001/tst.py (hunk 1 above)
Comment (ChristopherBittner) [reply 000/0001/tst.py]: Fixed.

**Review by EwaBittner — APPROVED** [review]

**Pull request merged**